cloverage/test/cloverage/source_test.clj

```diff
@@ -18,9 +18,10 @@
 
 (t/deftest multibyte-test
   (t/testing "Form reader reads files with multibyte chars"
-    (t/is (= (source/forms 'cloverage.sample.multibyte-sample)
-             '((ns cloverage.sample.multibyte-sample)
-               (def a "あ"))))))
+    (t/is (= '((ns cloverage.sample.multibyte-sample)
+               (def a "あ"))
+             (with-open [reader (source/form-reader 'cloverage.sample.multibyte-sample)]
+               (vec (source/forms reader)))))))
```

Comment:
The old `source/forms` function was only used by this test (and nowhere else); there was a different `instrument/source-forms` function in the instrumentation namespace. I consolidated both into a new `source/forms` function based on `instrument/source-forms`. I think a source forms function belongs in `source/` and not `instrument/` anyway. I've reworked this test to test the new version of the function.